pkg/commands/login.go

```diff
@@ -34,36 +32,22 @@ func RunLogin(ctx CommandContext, args LoginArgs) error {
 	}
 
 	cfg := &api.Config{
-		APIEndPoint:  "https://account.scaleway.com/",
-		Organization: strings.Trim(args.Organization, "\n"),
-		Token:        strings.Trim(args.Token, "\n"),
+		APIEndPoint:     "https://api.scaleway.com/",
+		ACCOUNTEndPoint: "https://account.scaleway.com/",
+		Organization:    strings.Trim(args.Organization, "\n"),
+		Token:           strings.Trim(args.Token, "\n"),
 	}
 
-	api, err := api.NewScalewayAPI(cfg.APIEndPoint, cfg.Organization, cfg.Token)
+	api, err := api.NewScalewayAPI(cfg.APIEndPoint, cfg.ACCOUNTEndPoint, cfg.Organization, cfg.Token)
 	if err != nil {
 		return fmt.Errorf("Unable to create ScalewayAPI: %s", err)
 	}
+	api.EnableAccountApi()
```

Comment:
Can you move this call in `api.CheckCredentials()`, and calls `DisableAccountApi` just after but still inside the same function ?